services/dauth-service/src/core/auth_vectors.rs

```diff
@@ -207,10 +207,18 @@ pub async fn next_backup_auth_vector(
         database::flood_vectors::get_first(&mut transaction, &av_request.user_id).await
     {
         vector = flood_row.to_auth_vector()?;
+
```

Comment:
This change looks good to me! One piece of meta-feedback is that especially for bug fixes like this, adding a sentence to the commit describing the bug at a high-level in english makes it a lot easier to review and more useful in the future if ever needing to look through the code history!